### DISCUSSION TITLE
Reanimated for react native 0.70.0-rc.2

### DISCUSSION THREAD

**Discussion by Stizz-wizz** [software-mansion/react-native-reanimated#3461] [Ideas]
WARNING reanimated - no version-specific reanimated AAR for react-native version 70 found.
Falling back to AAR for react-native version 69.
The react-native JSI interface is not ABI-safe yet, this may result in crashes.
Please post a pull request to implement support for react-native version 70 to the reanimated repo.
Thanks!

Is there a way to get reanimated to work with react native 0.70.0-rc.2 ?

**Comment by tomekzaw** [comment]
Hey, Reanimated 2.10.0 supports React Native 0.70.

**Reply by Stizz-wizz** [reply]
Great! I will test it out later. Thank you @tomekzaw

**Reply by hayder-nearsure** [reply]
It's not working for  0.70.3

**Reply by priyaWedo** [reply]
@tomekzaw 
i found this warning in my project and app get crash in react native 0.72 


WARNING reanimated - no version-specific reanimated AAR for react-native version 72 found.
Falling back to AAR for react-native version 71.
The react-native JSI interface is not ABI-safe yet, this may result in crashes.
Please post a pull request to implement support for react-native version 72 to the reanimated repo.
Thanks!

**Reply by tomekzaw** [reply]
@priyaWedo Congratulations!

**Reply by priyaWedo** [reply]
i created issue in react native issue 
https://github.com/facebook/react-native/issues/38751
